### PR TITLE
Fixed xContent dependencies due to OSCore changes

### PR DESCRIPTION
**Pull request by Angie-Zhang**
*Issue #, if available:*

https://github.com/opensearch-project/OpenSearch/issues/6841

*Description of changes:*
1. Bump version of 2.x to 2.7; 
2. Update gradle.build; 
3. Fix xContent dependencies change.

*CheckList:*
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/index-management/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Angie-Zhang** [comment]
Security test workflow fail as security plugin has not been updated yet.

**Review by AWSHurneyt — DISMISSED** [review]

File: build.gradle [hunk 1]
```diff
@@ -646,85 +604,51 @@ task integTestRemote(type: RestIntegTestTask)  {
...
+        'opensearch/plugins/opensearch-index-management-' + bwcVersion + '.zip'
 
-2.times {i ->
+2.times { i ->
     testClusters {
         "${baseName}$i" {
             testDistribution = "ARCHIVE"
+            versions = [bwcVersionShort, opensearch_version]
             numberOfNodes = 3
-            if (bwcBundleTest) {
```
Comment (bowenlan-amzn) [inline build.gradle:675]: Let's keep the bwc bundle test change, don't delete these

Comment (Angie-Zhang) [reply build.gradle:675]: Added back

**Comment by Angie-Zhang** [comment]
Multi node test workflow failed for known flaky tests.

**Review by AWSHurneyt — APPROVED** [review]

**Review by bowenlan-amzn — APPROVED** [review]
Thanks!

**Pull request merged**